proto/context/src/main/proto/spine/validation/field_views.proto

```diff
@@ -30,11 +30,8 @@ message RequiredField {
     // The error message template.
     string error_message = 2;
 
-    // `true` if the field is marked with `(required) = true`, `false` otherwise.
-    bool required = 3;
```

Comment:
If you delete this field, let's modify the documentation of its parent message to reflect the difference in the behavior. So from now on, the instances of `RequiredField` are only created for the declarations marked with `(required) = true`.